core/capabilities/ccip/delegate.go

```diff
@@ -251,10 +250,6 @@ func (d *Delegate) OnDeleteJob(ctx context.Context, spec job.Job) error {
 func (d *Delegate) getOCRKeys(ocrKeyBundleIDs job.JSONConfig) (map[string]ocr2key.KeyBundle, error) {
 	ocrKeys := make(map[string]ocr2key.KeyBundle)
 	for networkType, bundleIDRaw := range ocrKeyBundleIDs {
-		if networkType != relay.NetworkEVM {
-			return nil, fmt.Errorf("unsupported chain type: %s", networkType)
-		}
```

Comment:
This check can still be performed by the oracle creator using something like `_, supported := oraclecreator.plugins[networkType]`

Reply:
I've removed the check because it should be fine to declare OCR2 keys for chains that are currently unsupported, those keys will just be unused (a similar change is needed in transmitters). This way we won't even need to redeploy jobspecs when new chains with existing keys add CCIP support.

The jobspec code further down also works similarly: it simply exposes all available key types